this is https://github.com/ispc/ispc/commit/9b722217a2818bc4335bf2c7d3293d30aa5977b1 but they modified it for the release to disable this part
diff --git a/src/ispc_version.h b/src/ispc_version.h
index 0e28dfc..16660d8 100644
--- a/src/ispc_version.h
+++ b/src/ispc_version.h
@@ -51,7 +51,7 @@
 #define ISPC_LLVM_15_0 150000
 
 #define OLDEST_SUPPORTED_LLVM ISPC_LLVM_10_0
-#define LATEST_SUPPORTED_LLVM ISPC_LLVM_14_0
+#define LATEST_SUPPORTED_LLVM ISPC_LLVM_15_0
 
 #ifdef __ispc__xstr
 #undef __ispc__xstr
